make/targets/openshift/deps-gomod.mk

```diff
@@ -15,8 +15,7 @@ endef
 verify-deps: tmp_dir:=$(shell mktemp -d)
 verify-deps:
 	$(call restore-deps,$(tmp_dir))
-	$(deps_diff) "$(tmp_dir)"/{current,updated}/go.mod || echo '`go.mod` content is incorrect - did you run `go mod tidy`?' && false
-	$(deps_diff) "$(tmp_dir)"/{current,updated}/go.sum || echo '`go.sum` content is incorrect - did you run `go mod tidy`?' && false
```

Comment:
why removing the go.sum check?

Reply:
added it back, did not notice it was checking as different file, my bad